CONTRIBUTING.md

```diff
@@ -1,3 +1,169 @@
 # Guidelines for contributing
 
-Coming soon...
+## General Workflow
```

Comment:
I would move this harder, together with the first line on using Git, GitHub etc. and the Note below the "(Other) Communication" section.

I would only keep the Code of Conduct note at the very beginning, without any extra headers, so just after the `#Guidelines for contributing` header.

Then follow with "Submitting Issues", then "Other Communication", and only _then_ anything related to actually contributing under a single top-level (`##`) header (e.g., "Contributing Code Changes").

The reason is that submitting issues and reaching out on Discussions or via email is for a much wider audience, whereas the rest is only for the much smaller group of people who would actually like to contribute code.